src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/Hand.java

```diff
@@ -141,7 +141,7 @@ private static List<Hand> rotateHandsByLastWinner(
       Optional<Hand> winnerHand =
           sortedHands.stream().filter(hand -> hasHandWon(round, hand)).findFirst();
       if (winnerHand.isPresent()) {
-        int winnerHandIndex = sortedHands.indexOf(winnerHand.get());
+        int winnerHandIndex = sortedHands.indexOf(winnerHand.get()) * (-1);
         Collections.rotate(handsStartingWithPreviousWinner, winnerHandIndex);
```

Comment:
Der winnerHandIndex stimmt immer noch, man muss einfach -1 mal winnerHandIndex nehmen.
Das geht am einfachsten mit einem - vor der variable (glaube ich).

```suggestion
        int winnerHandIndex = sortedHands.indexOf(winnerHand.get());
        Collections.rotate(handsStartingWithPreviousWinner, -winnerHandIndex);
```

Reply:
Ah, noch besser, danke.